src/Dhl/Middleware/AuthMiddleware.php

```diff
@@ -92,6 +92,10 @@ class AuthMiddleware
         '/import-schemas/default' => ['GET'],
     ];
 
+    protected $authRequiredConfigRegexes = [
+        '/^(\/labels\/)?([0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12})/' => ['GET']
```

Comment:
betekend die `?` dat `/labels/` optioneel is?

Reply:
Nee toch, dat ie ook moet voldoen aan hetgeen ernaast?